vscode/src/circuit.ts

```diff
@@ -108,28 +125,35 @@ export async function showCircuitCommand(
 export function updateCircuitPanel(
   targetProfile: string,
   docPath: string,
-  circuitOrErrorHtml: CircuitData | string,
   reveal: boolean,
-  operation?: IOperationInfo | undefined,
+  params: {
+    circuit?: CircuitData;
+    errorHtml?: string;
+    simulating?: boolean;
+    operation?: IOperationInfo | undefined;
+  },
 ) {
   let title;
-  let subtitle;
-  if (operation) {
-    title = `${operation.operation} with ${operation.totalNumQubits} input qubits`;
-    subtitle = `Target profile: ${getTargetFriendlyName(targetProfile)} `;
+  let target;
+  if (params?.operation) {
+    title = `${params.operation.operation} with ${params.operation.totalNumQubits} input qubits`;
+    target = `Target profile: ${getTargetFriendlyName(targetProfile)} `;
```

Comment:
Looks like target is the same in both branches. Move it out of the if block and make it const.